### PR TITLE
New referenceValue parameter Create Payment edpt

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [X] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

PR created to publish the Release Notes "New referenceValue parameter in the Create Payment endpoint request body" (EDU-10947)

### CODE REVIEW

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+slug: "new-referencevalue-parameter-create-payment-endpoint"
+title: "New referenceValue parameter in the Create Payment endpoint request body"
+createdAt: 2023-10-24T00:00:00.000Z
+hidden: false
+type: "added"
+excerpt: "New parameter available in the Create Payment endpoint request body."
+---
+The `referenceValue` parameter was added to the Create Payment endpoint. This new field is mandatory and must be sent on the endpoint to represent the payment amount without interest applied. The values ​​of the `value` and `referenceValue` fields will be equal whenever the value of the `installmentsInterestRate` field is equal to zero. See the examples below:
```
Comment (PedroAntunesCosta) [inline docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md:9]:
```suggestion
The `referenceValue` parameter was added to the Create Payment endpoint request body. This new field is mandatory and must be sent on the endpoint to represent the payment amount without interest applied.

The values ​​of the `value` and `referenceValue` fields will be equal whenever the value of the `installmentsInterestRate` field is equal to zero.

See the examples below:
```

Comment (PedroAntunesCosta) [inline docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md:9]: Please add a link to the endpoint reference.

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md (hunk 1 above)
Comment (ricardoaerobr) [reply docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md]: Adicionado.

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+type: "added"
+excerpt: "New parameter available in the Create Payment endpoint request body."
+---
+The `referenceValue` parameter was added to the [Create Payment endpoint](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#post-/payments) request body. This new field is mandatory and must be sent on the endpoint to represent the payment amount without interest applied.
+
+The values ​​of the `value` and `referenceValue` fields will be equal whenever the value of the `installmentsInterestRate` field is equal to zero.
+
+See the examples below:
+
+- **Case 1**: Order value is 15.00 (interest free):
```
Comment (PedroAntunesCosta) [inline docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md:15]:
```suggestion
Example 1: Order value is 15.00 (interest free).
```

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  - `value`: 15.00
+  - `referenceValue`: 15.00
+  - `installments`: 1
+  - `installmentsInterestRate`: 0.00
+
+- **Case 2**: Order value is 15.00 (with interest of 1.00 per month, 3 installments):
```
Comment (PedroAntunesCosta) [inline docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md:22]:
```suggestion
Example 2: Order value is 15.00 (with interest of 1.00 per month, 3 installments).
```

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+---
+The `referenceValue` parameter was added to the [Create Payment endpoint](https://developers.vtex.com/docs/api-reference/payment-provider-protocol#post-/payments) request body. This new field is mandatory and must be sent on the endpoint to represent the payment amount without interest applied.
+
+The values ​​of the `value` and `referenceValue` fields will be equal whenever the value of the `installmentsInterestRate` field is equal to zero.
+
+See the examples below:
+
+- **Case 1**: Order value is 15.00 (interest free):
+
+  - `value`: 15.00
```
Comment (PedroAntunesCosta) [inline docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md:17]: This should be formatted as a request body example, not a list. If the body is too big, you can use `...` to represent the parts that don't matter to this release note.

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  - `referenceValue`: 15.00
+  - `installments`: 1
+  - `installmentsInterestRate`: 0.00
+
+- **Case 2**: Order value is 15.00 (with interest of 1.00 per month, 3 installments):
+
+  - `value`: 15.00
```
Comment (PedroAntunesCosta) [inline docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md:24]: This should be formatted as a request body example, not a list. If the body is too big, you can use `...` to represent the parts that don't matter to this release note.

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]
I just realized this is a release note. So it must contain at least a **What needs to be done?** heading, with precise instructions of how to adapt integrations and what will happen if they don't.

And maybe a **What changed?**

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md (hunk 4 above)
Comment (ricardoaerobr) [reply docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md]: Corrigido.

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md (hunk 5 above)
Comment (ricardoaerobr) [reply docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md]: Corrigido.

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]
Please add a `What needs to be done` section as suggested above.

Even if its not a breaking change, we must explicitly inform readers that "no action is needed" or maybe that "this field is already active without action needed. Whenever you use the XXXX request, you must include the field YYYY"

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+...
+"value": "15.00",
+"referenceValue": "12.00",
+"installments": 3,
+"installmentsInterestRate": "1.00"
+...
+```
+## What needs to be done?
+
+No action is need.
```
Comment (PedroAntunesCosta) [inline docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md:38]:
```suggestion
No action is needed.
```

File: docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+```json
+...
+"value": "15.00",
+"referenceValue": "12.00",
+"installments": 3,
+"installmentsInterestRate": "1.00"
+...
+```
+## What needs to be done?
```
Comment (PedroAntunesCosta) [inline docs/release-notes/new-referencevalue-parameter-create-payment-endpoint.md:36]:
```suggestion

## What needs to be done?
```

**Review by PedroAntunesCosta — APPROVED** [review]
LGTM, but please check suggestions

**Pull request merged**